Makefile

```diff
@@ -17,3 +17,18 @@ schema-docs:
 		-cat aipscan \
 		-s aipscan \
 		-debug
+
+build:
+	# Install Node modules if not present
+	if [ ! -d "node_modules" ]; then \
+		npm install; \
+	fi
+
+	# Bundle vendor assets
+	vite build -c build_config/base.config.js
```

Comment:
We should probably use `npx vite` instead to ensure that it's using the vite installed in this environment.

Reply:
Good call!